libraries/standard/http/utest/http_utest.c

```diff
@@ -204,79 +202,76 @@ static void setupRequestInfo( HTTPRequestInfo_t * pRequestInfo )
 static void setupBuffer( HTTPRequestHeaders_t * pRequestHeaders,
                          size_t bufferLen )
 {
-    pRequestHeaders->pBuffer = httpBuffer;
+    pRequestHeaders->pBuffer = testBuffer;
```

Comment:
What is the purpose of setting the input request buffer length the same as the expected buffer? Why not use the complete length of the buffer (i.e. `sizeof(testBuffer)`)?